test/suites/drift/suite_test.go

```diff
@@ -148,20 +148,87 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(node), &v1.Node{})).To(Succeed())
 		}).WithTimeout(time.Minute).Should(Succeed())
 	})
-})
+	It("should deprovision nodes that have drifted due to securitygroup", func() {
+		env.ExpectSettingsOverridden(map[string]string{
+			"featureGates.driftEnabled": "true",
+		})
+		By("creating a new securitygroup")
+		securitygroup := env.GetSecurityGroups(map[string]string{"karpenter.sh/discovery": settings.FromContext(env.Context).ClusterName})
+		Expect(len(securitygroup)).To(BeNumerically("==", 2))
+
+		createSecurityGroup := &ec2.CreateSecurityGroupInput{
+			GroupName:   awssdk.String("security-group-drift"),
+			Description: awssdk.String("Drift Test"),
```

Comment:
Since we're creating a resource here, we should be as descriptive about this so that we can easily identify this and clean this up later, if it leaks. 

Can you add something on top of this that infers something special about the environment and put it in the group name and in the description?
```suggestion
			Description: awssdk.String("End-to-end Drift Test"),
```